src/libs/actions/App.ts

```diff
@@ -43,8 +43,11 @@ Onyx.connect({
     },
 });
 
+// isSidebarLoaded isn't used directly when rendering the View. It's used in the event handler.
+// Using useOnyx would cause additional unnecessary rerenders that wouldn't change anything in the View,
+// so I'm leaving Onyx.connectWithoutView here.
```

Comment:
This comment is too lengthy and uses personal phrasing (“I’m”). Let's keep it more concise and neutral, just explaining the technical reasoning.

```suggestion
// `isSidebarLoaded` is only used inside the event handler, not during render.
// `useOnyx` would trigger extra rerenders without affecting the View, so `Onyx.connectWithoutView` is used instead
```